fluentforms/core/src/main/java/com/_4point/aem/fluentforms/impl/assembler/SafeDocAssemblerServiceAdapterWrapper.java

```diff
@@ -0,0 +1,49 @@
+package com._4point.aem.fluentforms.impl.assembler;
+
+import java.util.Map;
+import java.util.Objects;
+
+import com._4point.aem.fluentforms.api.Document;
+import com._4point.aem.fluentforms.api.assembler.AssemblerOptionsSpec;
+import com._4point.aem.fluentforms.api.assembler.AssemblerResult;
+import com._4point.aem.fluentforms.api.assembler.AssemblerService.AssemblerServiceException;
+import com.adobe.fd.assembler.client.OperationException;
+import com.adobe.fd.assembler.client.PDFAConversionOptionSpec;
+import com.adobe.fd.assembler.client.PDFAConversionResult;
+import com.adobe.fd.assembler.client.PDFAValidationOptionSpec;
+import com.adobe.fd.assembler.client.PDFAValidationResult;
+
+
+public class SafeDocAssemblerServiceAdapterWrapper implements TraditionalDocAssemblerService {
+	private final TraditionalDocAssemblerService assemblerService;
+
+	public SafeDocAssemblerServiceAdapterWrapper(TraditionalDocAssemblerService docAssemblerService) {
+		super();
+		this.assemblerService = docAssemblerService;
+	}
+
+	@Override
+	public AssemblerResult invoke(Document ddx, Map<String, Object> sourceDocuments,
+			AssemblerOptionsSpec adobAssemblerOptionSpec) throws AssemblerServiceException, OperationException {
+		Objects.requireNonNull(ddx, "ddx Document cannot be null.");
+		Objects.requireNonNull(sourceDocuments, "sourceDocuments cannot be null.");
```

Comment:
Again, null checks are already happening elsewhere.  We probably don't need to include them here/

Reply:
added null check for other methods of assemblerService